pytermgui/ansi_interface.py

```diff
@@ -723,114 +809,132 @@ def translate_mouse(code: str, method: str) -> list[MouseEvent | None] | None:
             "34": MouseAction.RIGHT_CLICK,
             "35": MouseAction.RELEASE,
             "64": MouseAction.LEFT_DRAG,
-            "66": MouseAction.RIGHT_CLICK,
+            "66": MouseAction.RIGHT_DRAG,
             "96": MouseAction.SCROLL_UP,
             "97": MouseAction.SCROLL_DOWN,
         },
     }
-
     mapping = mouse_codes[method]
     pattern = mapping["pattern"]
-
     events: list[MouseEvent | None] = []
     for sequence in code.split("\x1b"):
         if len(sequence) == 0:
             continue
-
         matches = list(pattern.finditer(sequence))
         if matches == []:
             return None
-
         for match in matches:
             identifier, *pos, release_code = match.groups()
 
             # decimal_xterm uses the last character's
             # capitalization to signify press/release state
             if len(release_code) > 0 and identifier == "0":
                 identifier += release_code
-
             if identifier in mapping:
                 action = mapping[identifier]
                 assert isinstance(action, MouseAction)
-
                 events.append(MouseEvent(action, (int(pos[0]), int(pos[1]))))
                 continue
-
             events.append(None)
```

Comment:
These linebreaks are there for readability. I do not agree with removing them.

Reply:
Ok